ios/RxBluetoothKit/RxCBPeripheral.swift

```diff
@@ -361,8 +361,13 @@ class RxCBPeripheral: RxPeripheralType {
         }
 
         @objc func peripheralIsReady(toSendWriteWithoutResponse peripheral: CBPeripheral) {
-            RxBluetoothKitLog.d("\(peripheral.logDescription) peripheralIsReady(toSendWriteWithoutResponse:\(peripheral.canSendWriteWithoutResponse)")
-            peripheralIsReadyToSendWriteWithoutResponseSubject.onNext(peripheral.canSendWriteWithoutResponse)
+            //resolve build errors with XCode 11 / iOS 13
+            var sendStatus = false
+            if #available(iOS 11.0, *) {
+              sendStatus = peripheral.canSendWriteWithoutResponse
+            }
+            RxBluetoothKitLog.d("\(peripheral.logDescription) peripheralIsReady(toSendWriteWithoutResponse:\(sendStatus)")
+            peripheralIsReadyToSendWriteWithoutResponseSubject.onNext(sendStatus)
```

Comment:
Can you modify your code to set canSendWriteWithoutResponse to true, when we are in 11.2 or below? We had similiar issue previously (check implementation of `canSendWriteWithoutResponse` inside RxCBPeripheral). Suggested change:
```swift
let canSendWriteWithoutResponse: Bool
if #available(iOS 11.2, *) {
    canSendWriteWithoutResponse = peripheral.canSendWriteWithoutResponse
} else {
    canSendWriteWithoutResponse = true
}
RxBluetoothKitLog.d("\(peripheral.logDescription) peripheralIsReady(toSendWriteWithoutResponse:\(canSendWriteWithoutResponse)")
peripheralIsReadyToSendWriteWithoutResponseSubject.onNext(canSendWriteWithoutResponse)
```

Reply:
ok, done

Reply:
Thank you :)